<launch>
	<!-- launches seds and ds nodes -->
	<node name="seds_node" pkg="seds" type="seds_node" />
	<node name="ds_node" pkg="seds" type="ds_node" />
	<node name="pr2_driver" pkg="seds" type="pr2_driver.py" />
</launch>